Testing/Tools/cxxtest/build_tools/SCons/cxxtest.py

```diff
@@ -61,7 +61,7 @@
 # the 1st argument to the function. This will result in the end executable
 # called that. Normal Program builder rules apply.
 #
-
+# ruff: noqa: F403   # Allow wild imports
 from SCons.Script import *
 from SCons.Builder import Builder
```

Comment:
Do you know what's going on here? Where does `SCons` come from?

Reply:
I have no idea if this file is actually used, or what it is used for! @thomashampson do you know?

I'm also not sure where `SCons` comes from (maybe this is an indication that it is unused)